config/locales/en.yml

```diff
@@ -242,4 +253,4 @@ en:
             slug:
               invalid: must be lowercase and can not contain spaces
   meta:
-    description: Forwarding the interest of Awesome in the universe, $1,000 at a time.
+    description: Forwarding the interest of Awesome in the universe, $1,000 at a time.
```

Comment:
Added. Just out of curiosity, is there a solid reason to prefer having newlines over not, or is it just convention? (Convention is totally fine; we don't do this at eSpark and I'm curious if there's a reason to.)